showkase-screenshot-testing-paparazzi-sample/src/test/java/com/airbnb/android/showkase/screenshot/testing/paparazzi/sample/MyPaparazziShowkaseScreenshotTest.kt

```diff
@@ -1,9 +1,36 @@
 package com.airbnb.android.showkase.screenshot.testing.paparazzi.sample
 
+import androidx.compose.ui.unit.LayoutDirection
+import app.cash.paparazzi.DeviceConfig
+import app.cash.paparazzi.Paparazzi
 import com.airbnb.android.showkase.annotation.ShowkaseScreenshot
+import com.airbnb.android.showkase.screenshot.testing.paparazzi.PaparazziShowkaseDeviceConfig
 import com.airbnb.android.showkase.screenshot.testing.paparazzi.PaparazziShowkaseScreenshotTest
+import com.airbnb.android.showkase.screenshot.testing.paparazzi.PaparazziShowkaseTestPreview
+import com.airbnb.android.showkase.screenshot.testing.paparazzi.PaparazziShowkaseUIMode
 
 @ShowkaseScreenshot(rootShowkaseClass = PaparazziSampleRootModule::class)
-abstract class MyPaparazziShowkaseScreenshotTest: PaparazziShowkaseScreenshotTest {
-    companion object: PaparazziShowkaseScreenshotTest.CompanionObject
+abstract class MyPaparazziShowkaseScreenshotTest : PaparazziShowkaseScreenshotTest {
+    override fun takePaparazziSnapshot(
+        paparazzi: Paparazzi,
+        testPreview: PaparazziShowkaseTestPreview,
+        direction: LayoutDirection,
+        mode: PaparazziShowkaseUIMode
+    ) {
+        if (paparazzi.resources.configuration.locales.get(0).toString() == "en-gb" ||
```

Comment:
Could you revert these changes. I'd like for this example to be the simplest version to make it easy for the users of the library. Good to verify that it can now support your use case though!

Reply:
Reverted the code specific to my use case